osu.Game/Overlays/Rankings/Tables/RankingsTable.cs

```diff
@@ -68,13 +70,20 @@ private void load()
 
         protected abstract Drawable[] CreateAdditionalContent(TModel item);
 
-        protected override Drawable CreateHeader(int index, TableColumn column) => new HeaderText(column?.Header ?? string.Empty, HighlightedColumn());
+        protected override Drawable CreateHeader(int index, TableColumn column)
+        {
+            var title = column?.Header ?? string.Empty;
+            var isHighlighted = HighlightedColumn() == title;
+            var isGrade = GradeColumns().Contains(title);
```

Comment:
I don't think these are acceptable. Leaving aside the mind-boggling design of `HighlightedColumn()` which existed previously, `GradeColumns()` is similarly strange.

* More idiomatic to have it as a property and not a method.
* More importantly, the premise that a generic rankings table should know what is or isn't a grade column is flawed and doing table-specific styling based on that is also flawed to me. Table-specific stylings should be applied in table-specific places. This implementation tries to be way too generic with what it does, therefore introducing 2 levels of inheritance for 4 usage cases and ending up being unreadable in the process:
```
RankingsTable
  CountriesTable
  UserBasedTable
    PerformanceTable
    ScoresTable
```

IMO we can't proceed with this code without de-generifying it.

Reply:
As an aside, the header colours are actually flipped right now:
![image](https://user-images.githubusercontent.com/20418176/76151438-71093f80-60b5-11ea-813d-0a89e5193f85.png)

Reply:
Moved the grade-column parts to `UserBasedTable` as that's where they're used (my bad for not noticing that in the first place), I'm far from being sure that this is the right way to do it, but I didn't have any better ideas.

Header colours fixed.